src/__tests__/imagings/ViewImagings.test.tsx

```diff
@@ -0,0 +1,156 @@
+import { Table } from '@hospitalrun/components'
+import { act } from '@testing-library/react'
+import { mount, ReactWrapper } from 'enzyme'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+import createMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import ViewImagings from '../../imagings/ViewImagings'
+import * as ButtonBarProvider from '../../page-header/button-toolbar/ButtonBarProvider'
+import * as titleUtil from '../../page-header/title/useTitle'
+import ImagingRepository from '../../shared/db/ImagingRepository'
+import Imaging from '../../shared/model/Imaging'
+import Permissions from '../../shared/model/Permissions'
+import { RootState } from '../../shared/store'
+
+const mockStore = createMockStore<RootState, any>([thunk])
+
+describe('View Imagings', () => {
+  describe('title', () => {
+    let titleSpy: any
+    beforeEach(async () => {
+      const store = mockStore({
+        title: '',
+        user: { permissions: [Permissions.ViewImagings, Permissions.RequestLab] },
+        imagings: { imagings: [] },
+      } as any)
+      titleSpy = jest.spyOn(titleUtil, 'default')
+      jest.spyOn(ImagingRepository, 'findAll').mockResolvedValue([])
+      await act(async () => {
+        await mount(
+          <Provider store={store}>
+            <Router history={createMemoryHistory()}>
+              <ViewImagings />
+            </Router>
+          </Provider>,
+        )
+      })
```

Comment:
lately, we have been putting this kind of setup in a `setup` function. I think we should continue this pattern since it makes it easy to make sure each test is not dependent on a previous test.

Each describe can share the `setup` function. i.e. one setup function for the whole test suite.